src/actions/followedAccounts.test.js

```diff
@@ -54,17 +54,23 @@ describe('actions: followedAccount', () => {
     expect(followedAccountRemoved(data)).to.be.deep.equal(expectedAction);
   });
 
-  it('should update a followed account if balance changed', () => {
+  it('should update a followed account if balance or privateKey changed', () => {
```

Comment:
This `it` is really bigger than it should. IMO each Case (1-5) should be a separate `it`.

I see that previously the first two cases were in the same `it`, even those should have been split.